pgtoolkit/ctl.py

```diff
@@ -182,18 +182,27 @@ def start(
         *,
         wait: Union[bool, int] = True,
         logfile: Optional[Union[Path, str]] = None,
+        **opts: Union[str, Literal[True]],
     ) -> CompletedProcess:
         """Start a PostgreSQL cluster.
 
         :param datadir: Path to database storage area
         :param wait: Wait until operation completes, if an integer value is
             passed, this will be used as --timeout value.
         :param logfile: Optional log file path
+        :param opts: extra options passed to ``postgres`` command.
+
+        Options name passed as `opts` should be underscore'd instead of dash'ed
+        and flag options should be passed a boolean ``True`` value; e.g.
+        ``F=True, work_meme=123`` for ``pg_ctl start -o '-F --work-meme=123'``.
```

Comment:
does `--work-meme` actually exist? I can see `-S WORK-MEM        set amount of memory for sorts (in kB)`.